ui/app/src/utils/xml.ts

```diff
@@ -181,3 +181,14 @@ export function deserialize(xml: string | Node, options?: X2jOptionsOptional): a
 export function cdataWrap(value: string): string {
   return `<![CDATA[${value}]]>`;
 }
+
+export function parseValidateDocument(content: string): XMLDocument | string {
```

Comment:
Is the parsed document ever used for something? I’m thinking the function should have a single responsibility.

Reply:
yes, the doc is used